pages/linux/ffuf.md

```diff
@@ -0,0 +1,24 @@
+# ffuf
+
+> A fast web fuzzer written in Go.
+> More information: <https://github.com/ffuf/ffuf>.
+
+- Directory discovery using a wordlist on a target website colorized and verbose:
+
+`ffuf -w {{/path/to/wordlist}} -u {{https://target/FUZZ}} -c -v`
+
+- Host-header fuzzing with host file on a traget website and match HTTP 200 responses:
+
+`ffuf -w {{hosts.txt}} -u {{https://example.org/}} -H "{{Host: FUZZ}}" -mc {{200}}`
```

Comment:
```suggestion
`ffuf -w {{path/to/hosts.txt}} -u {{https://example.org/}} -H "{{Host: FUZZ}}" -mc {{200}}`
```
Is the slash after `example.org` required in this case? If not I'd remove it.

Reply:
addressed in 4fe26e86dfcc04de1115ef613c87693a5e0b7333